doc/source/ray-core/tasks/using-ray-with-gpus.rst

```diff
@@ -123,4 +123,23 @@ This also lets the multi-node-type autoscaler know that there is demand for that
     :start-after: __accelerator_type_start__
     :end-before: __accelerator_type_end__
 
-See ``ray.util.accelerators`` for available accelerator types. Current automatically detected accelerator types include Nvidia GPUs.
+See ``ray.util.accelerators`` for available accelerator types. Current automatically detected accelerator types include Nvidia GPUs, AWS-neuron-cores.
```

Comment:
Would recommend making this a list so its easier to extend, and update the formatting? Something like:

```suggestion
See ``ray.util.accelerators`` for available accelerator types. Current automatically detected accelerator types include:

 - Nvidia GPUs
 - AWS Neuron Cores
```

Feel free to ignore the latter suggestion if there was some intention for the hyphens that I'm not aware of.